pages/_app.js

```diff
@@ -124,7 +141,7 @@ class MyApp extends App {
             );
 
         return (
-            <>
+            <PageContext.Provider value={{ isEditMode, nav, language }}>
                 <style global jsx>
                     {`
                         @import url('application/themes/travel/css/styles.dotsass');
```

Comment:
Check this out: https://github.com/dotCMS/dotcms-spa/pull/64/files#diff-13b175b22785184967bd1c87e42ec293R130